pagopa/io-app
# feat: [IOCOM-884] Green pass relevant screens, new DS

## Short description
This PR adapts the relevant Green Pass' screens to the new DS.
|Loading|Generic Error|Revoked|
|-|-|-|
|![Loading](https://github.com/pagopa/io-app/assets/5150343/3a6e7a03-b33a-4e91-b22c-51a893aa0263)|![GeneralError](https://github.com/pagopa/io-app/assets/5150343/0d582b53-f352-42fd-8675-5adbcb246925)|![Revoked](https://github.com/pagopa/io-app/assets/5150343/da91f806-6e5a-4dde-9a6b-e694bc479121)|

## List of changes proposed in this pull request
- All Green Pass' screens have been converted to functional components
- New DS on `EuCovidCertGenericErrorKoComponent` for a generic error while retrieving data
- New DS on `EuCovidCertLoadingScreen` for a loading Green Pass
- New DS on `EuCovidCertRevokedScreen` for a revoked Green Pass
- `Markdown` component now supports an input number of lines to display the loading skeleton

## How to test
Using the io-dev-api-server, generate messages for the Green Pass and check that all the screens listed above use the new DS.
Check that every other types (for Green Pass' messages) work properly, even if their screens do not conform to the new DS.
